src/features/content-explorer/content-explorer/ContentExplorer.scss

```diff
@@ -71,6 +71,31 @@
         }
     }
 
+    .bdl-ContentExplorerIncludeSubfolders {
+        display: inline-block;
+        width: 100%;
+        height: 8 * $bdl-grid-unit;
+        vertical-align: middle;
```

Comment:
`vertical-align` isn't that great with aligning. are we able to use flex-box instead?